src/libs/actions/App.js

```diff
@@ -128,6 +121,26 @@ function getPolicyIDList(policies) {
 function openApp(policies) {
     API.read('OpenApp', {
         policyIDList: getPolicyIDList(policies),
+    }, {
+        optimisticData: [{
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: ONYXKEYS.IS_LOADING_REPORT_DATA,
+            value: true,
+        }],
+        successData: [{
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: ONYXKEYS.INITIAL_REPORT_DATA_LOADED,
+            value: true,
```

Comment:
I'm trying to wrap my mind around how this key works... and I am kind of confused. When will it ever not be true?

1. I think the answer is "when the user signs out" - which is kind of weird. That's nearly the same thing as "an authToken exists in the session"
2. Does this need to be set to false in optimistic data? What about in failure data?
3. Does it matter that this is still being set to true in `fetchAllReports()`?

Reply:
>When will it ever not be true?

This key is not true on a first ever login or if the user signed out.

>Does this need to be set to false in optimistic data? What about in failure data?

The [default is false](https://github.com/Expensify/App/blob/751915e541bfad2352237ce9d6030413b23938f0/src/Expensify.js#L75). So I don't think we need to add it to either `optimisticData` nor `failureData`

>Does it matter that this is still being set to true in fetchAllReports()?

I don't think so. We are only calling `fetchAllReports` in `deletePolicy` now and that will be refactored as well.